pkgs/development/python-modules/beanquery/default.nix

```diff
@@ -7,18 +7,18 @@
   python-dateutil,
   pytestCheckHook,
   setuptools,
-  tatsu,
+  tatsu-lts,
 }:
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "beanquery";
-  version = "0.1.0";
+  version = "0.2.0-unstable-2025-01-10";
```

Comment:
Doesn't follow guidelines

Reply:
I believe it does.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

```
If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.
```

Reply:
Is there a 0.2.0?

Reply:
Changed to 0.1.0-unstable-2025-01-10